taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -140,7 +144,26 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
         }
       }
     });
-    setShownLanguages(allLanguageCodes);
+    // get shown languages from local storage if it exists else use all languages
```

Comment:
let's move this part on it's own useEffect

Reply:
I'll have to recalculate `allLanguageCodes` with the same `nodeObject` dependency

Reply:
the shownLanguages shouldn't depend on the `nodeObject`, but on the languages stored in the local storage. Having it this way means that it will depend on the `nodeObject` so, for example,  every time we reload the site it will override the stored values, same thing will happen if the nodeObject changes... so not the behavior we want. This should only be a representation of the localStorage values and be in sync with it